src/components/TheMap.vue

```diff
@@ -72,6 +76,27 @@ export default {
             this.$store.commit("app/updatingData", {
               updatingData: false
             });
+            updateSensorGeoJSON(map);
```

Comment:
We should move lines 79 - 99 before we `return getSensorData()` on line 75. There is a chance (no matter how small) that the `getSensorData` promise may resolve before we get to these lines and since we are returning, lines 79-99 will never get executed.

After you re-arrange the order, I think you can remove the explicit call to `updateSensorGeoJSON(map)` since when `updatingData` gets set to `false` on line 77, it should trigger a call to  `updateSensorGeoJSON(map)`.